docs/quick_start/starting.rst

```diff
@@ -18,13 +18,15 @@ For example, to start the Fledge system, open a session to the Fledge device and
 
 If authentication is enabled, which is the default mode for Fledge version 3.0 onward, then a number of the  commands require authentication. Authentication can be accomplished by several means;
 
-  - Set the environment variable *USERNAME* to be the user name.
+  - Set the environment variable *FLEDGE_USER* to be the user name.
     
   - Pass the *-u* flag flag to the command to specify a user name.
 
+  - Create an authentication file
+
   - If neither of the above are done the user will be prompted to enter a user name.
 
-In both cases the user will be prompted to enter a password. It is possible, but not recommended, to set an environment variable *PASSWORD* with the plain text version of the password.
+In both cases the user will be prompted to enter a password. It is possible, but not recommended, to set an environment variable *FLEDGE_PASSWORD* with the plain text version of the password.
```

Comment:
same as above

Reply:
As above, the previous documentation was wrong, it always was FLEDGE_PASSWORD.

`
if [ -z ${USERNAME+x} ]; then
        # If username is not set on the command line use environment variable if set
        if [ ! -z ${FLEDGE_USER+x} ]; then
                USERNAME=$FLEDGE_USER
        fi
fi

if [ -z ${PASSWORD+x} ]; then
        # If password is not set on the command line use environment variable if set
        if [ ! -z ${FLEDGE_PASSWORD+x} ]; then
                PASSWORD=$FLEDGE_PASSWORD
        fi
fi`

It appeared to work because it was using the internal name used within the script.